api/v1alpha1/contour_types.go

```diff
@@ -83,11 +87,37 @@ type NamespaceSpec struct {
 	RemoveOnDeletion bool `json:"removeOnDeletion,omitempty"`
 }
 
+const (
+	// Available indicates that the contour is running and available.
+	ContourAvailableConditionType = "Available"
+)
+
 // ContourStatus defines the observed state of Contour.
 type ContourStatus struct {
-	// availableReplicas is the number of observed available Contour replicas
-	// according to the deployment.
+	// availableReplicas is the number of observed available Contour
+	// replicas according to the deployment. The deployment and its
+	// pods will reside in the namespace specified in spec.namespace.name
+	// of the contour.
 	AvailableReplicas int32 `json:"availableReplicas"`
+
+	// availablePods is the number of observed available Envoy pods.
+	// The daemonset and its pods will reside in the namespace specified
+	// in spec.namespace.name of the contour.
+	AvailablePods int32 `json:"availablePods"`
```

Comment:
Personally I do think it'd be clearer if these two field names differentiated between Contour pods and Envoy pods - e.g. `AvailableContourPods` and `AvailableEnvoyPods` or something like that.